doc/source/operations/rabbitmq.rst

```diff
@@ -0,0 +1,90 @@
+===============================
+Rolling out changes to RabbitMQ
+===============================
+
+Prior to upgrading, some changes need to be rolled out for RabbitMQ.
+This should make RabbitMQ more stable during the upgrade(s). These changes
+are:
+
+* Enable the high availability setting via Kolla-Ansible
+  ``om_enable_rabbitmq_high_availability``.
+* Update RabbitMQ to version 3.9.22
```

Comment:
Need a bit more info on when to do this (wallaby & xena if running old images), and how to do so.